aspnetcore/razor-components/components.md

```diff
@@ -602,7 +603,7 @@ public class ThemeInfo
 }
 ```
 
-An ancestor component can provide a cascading value using the `CascadingValue` component. The `CascadingValue` component wraps a subtree of the component hierarchy and supplies a single value to all components within that subtree.
+An ancestor component can provide a cascading value using the Cascading Value component. The Cascading Value component wraps a subtree of the component hierarchy and supplies a single value to all components within that subtree.
```

Comment:
Components are classes, so I think making their names as code is appropriate, but I'm fine with this too.

Reply:
We're not consistent about it. For controllers and views, we frequently show ...

* Home controller
* About view

... but there are many appearances of ...

* `HomeController` (especially referring directly to the class, which makes sense)
* `About` view

... so to the team: Let's not open a new issue to address the inconsistency but just note your guidance for new topics and topic overhauls. I prefer for general reference ...

* Home controller
* About view 
* Cascading Value component (over "`CascadingValue` component" unless referring to the `CascadingValue` component **_class_** specifically)

... and go with code style when referring to a class, view model, or action method.

Our inline code styling is kind'a ... well ... *blocky*-looking to me (to the point of even possibly slowing down reading and comprehension, i.e., it seems a little distracting imo). See: https://github.com/aspnet/Docs/issues/4657 where I play with alternative styles. I realize that you probably had a number of internal high-level meetings on doc styles. *Just say'in!* :smile:

Reply:
I'm OK with this approach but I'd like @Rick-Anderson and @scottaddie to weigh in.

Reply:
Hit me up later if you decide to go another way. Plenty of time to do something different for the Preview 3 pass.